lab/environments.py

```diff
@@ -391,9 +392,13 @@ def __init__(
         self.qos = qos
         self.time_limit_per_task = time_limit_per_task
         self.memory_per_cpu = memory_per_cpu
+        self.cpus_per_task = cpus_per_task
         self.export = export
         self.setup = setup
 
+        if cpus_per_task:
+            self.extra_options += f"\n#SBATCH --cpus-per-task={cpus_per_task}"
```

Comment:
To be more in line with the other options, please add this line to the template file and pass cpus_per_task when the template is filled.